app/controllers/concerns/verify_sp_attributes_concern.rb

```diff
@@ -28,6 +28,7 @@ def update_verified_attributes
       verified_attributes: sp_session[:requested_attributes],
       last_consented_at: Time.zone.now,
       clear_deleted_at: true,
+      email_address_id: session[:sp_email],
```

Comment:
to match other change
```suggestion
      email_address_id: session[:sp_email_id],
```